app/views/crimes/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Editing crime</h1>
```

Comment:
Change to "Edit Crime" or use interpolation with <h1>Edit <%= @crime.crime_type %></h1>